### DISCUSSION TITLE
Copilot inline chat : can not type in inline-chat window (Visual Studio 2022 Korean)

### DISCUSSION THREAD

**Discussion by hg98-park** [community/community#110368] [Copilot Conversations]
### Select Topic Area

Bug

### Body

When I press 'alt+/' to activate the copilot inline chat, the inline chat window is displayed but it doesn't accept my input. 
The cursor stays at the last position and when I click inside the inline chat window the cursor does not appear inside the window.
When I push the 'send' button it is not clicked.
I have Github Copilot Business plan and the Copilot Chat Pane is working fine. Only inline chat window have this problem.

My Visual Studio 2022 info :
Microsoft Visual Studio Professional 2022
버전 17.8.5
VisualStudio.17.Release/17.8.5+34511.84
Microsoft .NET Framework
버전 4.8.04084

설치된 버전: Professional

Visual C++ 2022   00483-10804-13865-AA825
Microsoft Visual C++ 2022

ASP.NET and Web Tools   17.8.358.6298
ASP.NET and Web Tools

Azure App Service 도구 v3.0.0   17.8.358.6298
Azure App Service 도구 v3.0.0

C# 도구   4.8.0-7.23572.1+7b75981cf3bd520b86ec4ed00ec156c8bc48e4eb
IDE에서 사용되는 C# 구성 요소입니다. 프로젝트 형식 및 설정에 따라 다른 버전의 컴파일러를 사용할 수 있습니다.

GitHub Copilot   1.156.0.0 (v1.156.0.0@82fafb995)
GitHub Copilot is an AI pair programmer that helps you write code faster and with less work.

GitHub Copilot Agent   1.156.0

Microsoft JVM Debugger   1.0
Provides support for connecting the Visual Studio debugger to JDWP compatible Java Virtual Machines

NuGet 패키지 관리자   6.8.0
Visual Studio의 NuGet 패키지 관리자입니다. NuGet에 대한 자세한 내용은 https://docs.nuget.org/를 참조하세요.

Test Adapter for Boost.Test   1.0
Boost.Test용으로 작성된 단위 테스트를 사용하여 Visual Studio의 테스트 도구를 사용하도록 설정합니다. 사용 약관 및 타사 통지는 확장 설치 디렉터리에서 확인할 수 있습니다.

Test Adapter for Google Test   1.0
Google Test용으로 작성된 단위 테스트와 함께 Visual Studio의 테스트 도구를 사용합니다. 사용 약관 및 타사 고지 사항은 확장 설치 디렉터리에서 확인할 수 있습니다.

TypeScript 도구   17.0.20920.2001
Microsoft Visual Studio용 TypeScript 도구

Visual Assist   10.9.2511.0
For more information about Visual Assist, see the Whole Tomato Software website at http://www.WholeTomato.com. Copyright (c)1997-2023 Whole Tomato Software, LLC

Visual Basic 도구   4.8.0-7.23572.1+7b75981cf3bd520b86ec4ed00ec156c8bc48e4eb
IDE에서 사용되는 Visual Basic 구성 요소입니다. 프로젝트 형식 및 설정에 따라 다른 버전의 컴파일러를 사용할 수 있습니다.

Visual F# Tools   17.8.0-beta.23475.2+10f956e631a1efc0f7f5e49c626c494cd32b1f50
Microsoft Visual F# Tools

Visual Studio IntelliCode   2.2
Visual Studio 위한 AI 지원 개발.

**Comment by ayim** [comment]
Hi! Thanks for reporting this issue and being so detailed with the error report information.
Could you please report this issue using the VS Feedback mechanism so that we can have additional telemetry and environment information to better debug this for you?

![image](https://github.com/community/community/assets/10951869/30d599b7-83a8-402f-a107-f4745251fae2)

--
Aaron Yim
Product Manager for GitHub Copilot in Visual Studio 2022
github:ayim | aayim@microsoft.com

**Reply by hg98-park** [reply]
Thanks for the feedback.
I pushed 'Report a Problem...' button and the following screen showed.
![image](https://github.com/community/community/assets/63629521/b108ed08-3269-45f4-86d1-d7910d71dcc6)

press 'x'  then
![image](https://github.com/community/community/assets/63629521/c4dd5a63-ddb4-409e-9501-01f489230a95)

My PC is behind the company firewall.

**Reply by ayim** [reply]
@hg98-park could you try clicking "Report a Problem" a few more times? It sometimes takes a few tries for me as well.

**Reply by hg98-park** [reply]
I tried 10 times and all failed. Here's the error message while tried ::
(The korean word saids there's something wrong with the certification)

Unexpected error
ERR_UNHANDLED: System.Net.Http.HttpRequestException: 이 요청을 보내는 동안 오류가 발생했습니다. ---> System.Net.WebException: 기본 연결이 닫혔습니다. SSL/TLS 보안 채널에 대한 트러스트 관계를 설정할 수 없습니다. ---> System.Security.Authentication.AuthenticationException: 유효성 검사 절차에 따르면 원격 인증서가 잘못되었습니다. 위치: System.Net.TlsStream.EndWrite(IAsyncResult asyncResult) 위치: System.Net.PooledStream.EndWrite(IAsyncResult asyncResult) 위치: System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar) --- 내부 예외 스택 추적의 끝 --- 위치: System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult) 위치: System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar) --- 내부 예외 스택 추적의 끝 --- 위치: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) 위치: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 위치: Feedback.HttpHelper.<SendAsync>d__1.MoveNext() --- 예외가 throw된 이전 위치의 스택 추적 끝 --- 위치: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) 위치: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 위치: Feedback.Communication.CommandHandler.<SendAsync>d__64.MoveNext() --- 예외가 throw된 이전 위치의 스택 추적 끝 --- 위치: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) 위치: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 위치: Feedback.Communication.CommandHandler.<LoadApprovedFileExtensionsAsync>d__19.MoveNext() --- 예외가 throw된 이전 위치의 스택 추적 끝 --- 위치: System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_0(Object state) 위치: System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs) 위치: System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)

Unexpected error ERR_UNHANDLED: System.Net.Http.HttpRequestException: 이 요청을 보내는 동안 오류가 발생했습니다. ---> System.Net.WebException: 기본 연결이 닫혔습니다. SSL/TLS 보안 채널에 대한 트러스트 관계를 설정할 수 없습니다. ---> System.Security.Authentication.AuthenticationException: 유효성 검사 절차에 따르면 원격 인증서가 잘못되었습니다. 위치: System.Net.TlsStream.EndWrite(IAsyncResult asyncResult) 위치: System.Net.PooledStream.EndWrite(IAsyncResult asyncResult) 위치: System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar) --- 내부 예외 스택 추적의 끝 --- 위치: System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult) 위치: System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar) --- 내부 예외 스택 추적의 끝 --- 위치: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) 위치: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 위치: Feedback.HttpHelper.<SendAsync>d__1.MoveNext() --- 예외가 throw된 이전 위치의 스택 추적 끝 --- 위치: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) 위치: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 위치: Feedback.Communication.CommandHandler.<SendAsync>d__64.MoveNext() --- 예외가 throw된 이전 위치의 스택 추적 끝 --- 위치: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) 위치: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 위치: Feedback.Communication.CommandHandler.<LoadApprovedFileExtensionsAsync>d__19.MoveNext() --- 예외가 throw된 이전 위치의 스택 추적 끝 --- 위치: System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_0(Object state) 위치: System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs) 위치: System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)

**Reply by hg98-park** [reply]
FYI I tried my co-worker's PC and same problem. Inline chat is not working.

**Reply by ayim** [reply]
I filed a feedback ticket here for you so the team can get started.
Please keep an eye to see if we have any updates or need more information as I will be on vacation for the next week.

**Comment by noellelc** [comment]
Thank you for reporting this. There is an open issue that is under investigation that you can follow here: https://developercommunity.visualstudio.com/t/Copilot-inline-chat-:-can-not-type-in-in/10607423

**Comment by pgroene** [comment]
@hg98-park if you are able to attach copilot logs to the feedback ticket that @noellelc  did mention above, I'll be able to investigate what the issue is you are facing. From the info you provided till now I can only see that the Chat extension is not reported as installed. The logs will also provide me with the Infomation I need to look at all your Visual Studio telemetry and identify any issues there. You can find the logs in %temp%/VSGitHubCopilotLogs

**Reply by hg98-park** [reply]
Thnaks for the quick reply.
I just send copilot chat log to the feedback ticket with ID as phg98.

**Comment by gokhan-or** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> When I press 'alt+/' to activate the copilot inline chat, the inline chat window is displayed but it doesn't accept my input. 
> The cursor stays at the last position and when I click inside the inline chat window the cursor does not appear inside the window.
> When I push the 'send' button it is not clicked.
> I have Github Copilot Business plan and the Copilot Chat Pane is working fine. Only inline chat window have this problem.
> 
> My Visual Studio 2022 info :
> Microsoft Visual Studio Professional 2022
> 버전 17.8.5
> VisualStudio.17.Release/17.8.5+34511.84
> Microsoft .NET Framework
> 버전 4.8.04084
> 
> 설치된 버전: Professional
> 
> Visual C++ 2022   00483-10804-13865-AA825
> Microsoft Visual C++ 2022
> 
> ASP.NET and Web Tools   17.8.358.6298
> ASP.NET and Web Tools
> 
> Azure App Service 도구 v3.0.0   17.8.358.6298
> Azure App Service 도구 v3.0.0
> 
> C# 도구   4.8.0-7.23572.1+7b75981cf3bd520b86ec4ed00ec156c8bc48e4eb
> IDE에서 사용되는 C# 구성 요소입니다. 프로젝트 형식 및 설정에 따라 다른 버전의 컴파일러를 사용할 수 있습니다.
> 
> GitHub Copilot   1.156.0.0 (v1.156.0.0@82fafb995)
> GitHub Copilot is an AI pair programmer that helps you write code faster and with less work.
> 
> GitHub Copilot Agent   1.156.0
> 
> Microsoft JVM Debugger   1.0
> Provides support for connecting the Visual Studio debugger to JDWP compatible Java Virtual Machines
> 
> NuGet 패키지 관리자   6.8.0
> Visual Studio의 NuGet 패키지 관리자입니다. NuGet에 대한 자세한 내용은 https://docs.nuget.org/를 참조하세요.
> 
> Test Adapter for Boost.Test   1.0
> Boost.Test용으로 작성된 단위 테스트를 사용하여 Visual Studio의 테스트 도구를 사용하도록 설정합니다. 사용 약관 및 타사 통지는 확장 설치 디렉터리에서 확인할 수 있습니다.
> 
> Test Adapter for Google Test   1.0
> Google Test용으로 작성된 단위 테스트와 함께 Visual Studio의 테스트 도구를 사용합니다. 사용 약관 및 타사 고지 사항은 확장 설치 디렉터리에서 확인할 수 있습니다.
> 
> TypeScript 도구   17.0.20920.2001
> Microsoft Visual Studio용 TypeScript 도구
> 
> Visual Assist   10.9.2511.0
> For more information about Visual Assist, see the Whole Tomato Software website at http://www.WholeTomato.com. Copyright (c)1997-2023 Whole Tomato Software, LLC
> 
> Visual Basic 도구   4.8.0-7.23572.1+7b75981cf3bd520b86ec4ed00ec156c8bc48e4eb
> IDE에서 사용되는 Visual Basic 구성 요소입니다. 프로젝트 형식 및 설정에 따라 다른 버전의 컴파일러를 사용할 수 있습니다.
> 
> Visual F# Tools   17.8.0-beta.23475.2+10f956e631a1efc0f7f5e49c626c494cd32b1f50
> Microsoft Visual F# Tools
> 
> Visual Studio IntelliCode   2.2
> Visual Studio 위한 AI 지원 개발.

**Reply by hg98-park** [reply]
I can not see the text you wrote. Only my previous text is shown.

**Comment by LoadStar822** [comment]
Have you solved this problem? I am currently facing the same problem

**Comment by hg98-park** [comment]
They said upgrade the Visual Assist will solve the problem although I hadn’t tried it because I quit the Copilot.
Hope this helps.

**Reply by LoadStar822** [reply]
I resolved this issue by disabling Visual Assist, thank you very much for your help!
